packages/react-components/src/components/VideoGallery/OverflowGallery.tsx

```diff
@@ -114,8 +118,13 @@ export const OverflowGallery = (props: {
         horizontalGalleryElements={overflowGalleryElements ? overflowGalleryElements : [<></>]}
```

Comment:
Consider instead of showing a ScrollableHorizontalGallery when `isNarrow` instead expose a property `overflowGalleryPagination: 'infiniteScroll' | 'paginated'`

Reply:
That is something we should do but I will leave it for another PR because that would cause an API change.